js/Grid/Paging.md

```diff
@@ -1,13 +1,13 @@
 ---
 layout: post
-title: Paging with Grid widget for Syncfusion Essential JS
-description: How to enable paging and its functionalites.
+title: Paging in JavaScript Grid control | Syncfusion
+description: Learn about paging support in Syncfusion Essential JavaScript Grid control, its elements and more.
```

Comment:
Change like this - Learn here about [check and change in all places]

Reply:
changed